<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BobuxNet Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="TopBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="accessibleName">
       <string>TopBar</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame[accessibleName=&quot;TopBar&quot;] {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.983, y2:0.954545, stop:0 rgba(142, 187, 255, 255), stop:1 rgba(170, 255, 255, 255));
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Money">
         <property name="text">
          <string>0 BOBUX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="AddressBar">
         <property name="text">
          <string>hello.com</string>
         </property>
         <property name="placeholderText">
          <string>Shop Address..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NavigateButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>Navigate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="WebContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>736</width>
         <height>311</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="WebBodyLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPage">
    <property name="title">
     <string>Page</string>
    </property>
    <addaction name="actionReturn"/>
   </widget>
   <addaction name="menuPage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReturn">
   <property name="text">
    <string>Return</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
